rezervo/utils/cron_utils.py

```diff
@@ -74,37 +73,28 @@ def build_cron_job_for_class(
         *generate_booking_schedule(
             _class,
             cron_config,
-            get_integration(integration).booking_open_days_before_class,
             precheck,
         )
     )
     return j
 
 
-def generate_booking_schedule(
-    _class: Class, cron_config: Cron, booking_open_days_before: int, precheck: bool
-):
+def generate_booking_schedule(_class: Class, cron_config: Cron, precheck: bool):
+    opening_time = datetime.fromisoformat(_class.booking_opens_at)
+
     # Using current datetime simply as a starting point
     # We really only care about the "wall clock" part, which is replaced by input values
-    activity_time = datetime.datetime.now().replace(
-        hour=_class.time.hour,
-        minute=_class.time.minute,
+    activity_time = datetime.now().replace(
```

Comment:
Don't think this is correct, as it will now use only the hour and minute from the `booking_opens_at`. I think `activity_time` should be replaced by `opening_time`.

Reply:
Nvm, it uses the correct weekday further down. Still, no point in keeping `activity_time` now that we have a proper datetime for the booking opening.

Reply:
However... the precheck block uses `booking_time.weekday()`, which is based on the current date, so it still might cause issues. So yeah, `activity_time` should leave 🚀